indico/core/storage/models.py

```diff
@@ -136,6 +136,19 @@ def size(cls):
         )
 
     @declared_attr
+    def md5(cls):
+        """
+        An MD5 hash of the file.
+
+        Automatically assigned when `save()` is called.
+        """
+        return db.Column(
+            db.String(32),
```

Comment:
https://stackoverflow.com/questions/4848964/postgresql-difference-between-text-and-varchar-character-varying is pretty interesting about postgres text types